test/functional/page_objects/dashboard_page.js

```diff
@@ -549,59 +532,73 @@ export function DashboardPageProvider({ getService, getPageObjects }) {
-        return slices[0].click();
-      });
+    async filterOnPieSlice(sliceValue) {
+      log.debug(`Filtering on a pie slice with optional value ${sliceValue}`);
+      if (sliceValue) {
+        await testSubjects.click(`pieSlice-${sliceValue}`);
+      } else {
+        await retry.try(async () => {
+          const slices = await find.allByCssSelector('svg > g > g.arcs > path.slice');
+          log.debug('Slices found:' + slices.length);
+          return slices[0].click();
+        });
+      }
     }
 
-    async arePanelMainMenuOptionsOpen(panel) {
+    async arePanelMainMenuOptionsOpen(parent) {
       log.debug('arePanelMainMenuOptionsOpen');
       // Sub menu used arbitrarily - any option on the main menu panel would do.
-      return panel ?
-        await testSubjects.descendantExists('dashboardPanelOptionsSubMenuLink', panel) :
+      return parent ?
+        await testSubjects.descendantExists('dashboardPanelOptionsSubMenuLink', parent) :
         await testSubjects.exists('dashboardPanelOptionsSubMenuLink');
     }
 
-    async openPanelOptions(panel) {
+    async openPanelOptions(parent) {
       log.debug('openPanelOptions');
-      const panelOpen = await this.arePanelMainMenuOptionsOpen(panel);
+      const panelOpen = await this.arePanelMainMenuOptionsOpen(parent);
       if (!panelOpen) {
         await retry.try(async () => {
-          await (panel ? remote.moveMouseTo(panel) : testSubjects.moveMouseTo('dashboardPanelTitle'));
-          const toggleMenuItem = panel ?
-            await testSubjects.findDescendant('dashboardPanelToggleMenuIcon', panel) :
+          await (parent ? remote.moveMouseTo(parent) : testSubjects.moveMouseTo('dashboardPanelTitle'));
+          const toggleMenuItem = parent ?
+            await testSubjects.findDescendant('dashboardPanelToggleMenuIcon', parent) :
             await testSubjects.find('dashboardPanelToggleMenuIcon');
           await toggleMenuItem.click();
-          const panelOpen = await this.arePanelMainMenuOptionsOpen(panel);
+          const panelOpen = await this.arePanelMainMenuOptionsOpen(parent);
           if (!panelOpen) { throw new Error('Panel menu still not open'); }
         });
       }
     }
 
-    async toggleExpandPanel(panel) {
-      await (panel ? remote.moveMouseTo(panel) : testSubjects.moveMouseTo('dashboardPanelTitle'));
+    async toggleExpandPanel(parent) {
+      await (parent ? remote.moveMouseTo(parent) : testSubjects.moveMouseTo('dashboardPanelTitle'));
       const expandShown = await testSubjects.exists('dashboardPanelExpandIcon');
       if (!expandShown) {
-        await this.openPanelOptions(panel);
+        await this.openPanelOptions(parent);
       }
       await testSubjects.click('dashboardPanelExpandIcon');
     }
 
-    async setCustomPanelTitle(customTitle, panel) {
-      log.debug(`setCustomPanelTitle(${customTitle}, ${panel})`);
-      await this.openPanelOptions(panel);
+    /**
```

Comment:
Honestly, this is the only place in this entire file that jsdoc comments are used. I'd personally get rid of them, and just add a descriptive `//` comment if you think one is necessary. Buuuut that's maybe terrible advice.

Reply:
As we move towards typescript, these kinds of comments will become more prolific.  This does look inconsistent, but if the function deserves a comment (I thought it did, `originalTitle` doesn't obviously reflect that it's being used to find the panel to change, and that it's optional), I think it might as well be jsdoc vs `//`.  Appreciate the feedback, but I think I'll leave this one here.